x-pack/plugins/aiops/public/components/log_rate_analysis_results_table/log_rate_analysis_results_table.tsx

```diff
@@ -83,6 +83,11 @@ export const LogRateAnalysisResultsTable: FC<LogRateAnalysisResultsTableProps> =
   const selectedSignificantItem = useAppSelector(
     (s) => s.logRateAnalysisTableRow.selectedSignificantItem
   );
+  const {
+    analysisType,
+    windowParameters,
+    documentStats: { documentCountStats },
+  } = useAppSelector((s) => s.logRateAnalysis);
```

Comment:
Since `useColumns` is a hook, you could move this inside the hook itself so doesn't need to get passed on as arguments to `useColumns`.